docs/source/changelog.rst

```diff
@@ -24,7 +24,7 @@ New Features
 Bugfixes
 ========
 
-- There is nothing here yet, check back later.
+#. Add ``disable_ligatures`` to Text ( :class:`Text` ). (via :pr:`804`)
```

Comment:
@behackl What should I do in this changelog?

Reply:
Ideally: move your line from here to the wiki (https://github.com/ManimCommunity/manim/wiki/Changelog-for-next-release) and don't change the changelog here (in order to avoid a conflict with #822).

Reply:
Or let it be from next PR not this.

Reply:
Or... move your line from here to the wiki :sweat_smile:

Reply:
Done.